core/src/main/java/org/elasticsearch/cluster/routing/allocation/decider/Decision.java

```diff
@@ -316,6 +321,13 @@ public String label() {
         }
 
         @Override
+        @Nullable
+        public String getExplanation() {
+            // multi-decisions have no explanation
+            return null;
```

Comment:
Should multi-decisions not be a concatenation of their sub-decision explanations?